### PR TITLE
Fixes Lightning CI

### PR DESCRIPTION
**Pull request by PiotrJander**

### CODE REVIEW

File: integrations/pytorch-lightning/Neptune-PyTorch-Lightning-advanced.ipynb [hunk 1]
```diff
@@ -63,7 +63,7 @@
    },
    "outputs": [],
    "source": [
-    "! pip install pytorch-lightning==1.0.0 neptune-client==0.4.123 torch==1.6.0 torchvision==0.7.0"
+    "! pip install pytorch-lightning==1.0.0 neptune-client==0.4.123 torch==1.6.0 torchvision==0.7.0 -f https://download.pytorch.org/whl/torch_stable.html"
```
Comment (jakubczakon) [inline integrations/pytorch-lightning/Neptune-PyTorch-Lightning-advanced.ipynb:66]: @PiotrJander what does -f  mean here?
Should we still keep torch==1.6.0 in there?

Comment (PiotrJander) [reply integrations/pytorch-lightning/Neptune-PyTorch-Lightning-advanced.ipynb:66]: Apparently the canonical pip command for installing torch is here https://pytorch.org/get-started/locally/ and for Windows it includes the `-f https://download.pytorch.org/whl/torch_stable.html` option. It doesn't seem to hurt on other platforms.

**Review by PiotrJander — COMMENTED** [review]
If you look at Actions, we made progress on Windows as Torch now gets installed: https://github.com/neptune-ai/neptune-examples/pull/12/checks?check_run_id=1281111884 .

But Windows runs still fail with 

```
  File "D:\a\neptune-examples\neptune-examples\integrations\pytorch-lightning\tests\Neptune-PyTorch-Lightning-advanced.py", line 7, in <module>
    get_ipython().system(' pip install pytorch-lightning==1.0.0 neptune-client==0.4.123 torch==1.6.0 torchvision==0.7.0 -f https://download.pytorch.org/whl/torch_stable.html')
NameError: name 'get_ipython' is not defined
```

and I've no idea why.

**Comment by PiotrJander** [comment]
Also, MacOS with Python 3.8 fails, and it also fails on my own Mac with Python 3.8 (!!), so it's not just a CI issue.

The error is 

```
~/projects/neptune-examples/.env/lib/python3.8/site-packages/torch/utils/data/dataloader.py in _try_get_data(self, timeout)
    790             if len(failed_workers) > 0:
    791                 pids_str = ', '.join(str(w.pid) for w in failed_workers)
--> 792                 raise RuntimeError('DataLoader worker (pid(s) {}) exited unexpectedly'.format(pids_str))
    793             if isinstance(e, queue.Empty):
    794                 return (False, None)

RuntimeError: DataLoader worker (pid(s) 70397) exited unexpectedly
```

File: integrations/pytorch-lightning/Neptune-PyTorch-Lightning-advanced.ipynb [hunk 2]
```diff
@@ -195,7 +195,7 @@
     "                          'decay_factor': 0.95}\n",
     "\n",
     "LightningDataModule_Params = {'batch_size': 32,\n",
-    "                              'num_workers': 4,\n",
+    "                              'num_workers': 0,\n",
```
Comment (PiotrJander) [inline integrations/pytorch-lightning/Neptune-PyTorch-Lightning-advanced.ipynb:198]: I followed the advice here https://github.com/pytorch/pytorch/issues/5301 and it fixed Lightning with Python 3.8 on a Mac, but not sure if performance is as good.

I hope this is fine - we want to CI to pass, even if it takes 2 minutes longer. Plus this is not Neptune's problem but Lightning's.

**Review by jakubczakon — APPROVED** [review]

**Pull request merged**